package.json

```diff
@@ -28,6 +28,7 @@
     "node-fetch": "^2.1.1",
     "node-sass-chokidar": "^0.0.3",
     "prop-types": "^15.5.10",
+    "query-string": "^6.2.0",
```

Comment:
I don't think you need to add dependency. Just use the react-router param match function

Reply:
Unless this does something more but looking at the code you don't need it at all and it will actually be simpler